example/rest-api/art-institute/inc/queries/class-art-institute-datasource.php

```diff
@@ -1,17 +0,0 @@
-<?php
-
-namespace RemoteDataBlocks\Example\ArtInstituteOfChicago;
-
-use RemoteDataBlocks\Config\HttpDatasource;
-
-class ArtInstituteOfChicagoDatasource extends HttpDatasource {
```

Comment:
These examples are now drifting from simple exercises that help the implementor. This feels like a step backwards for example code. Or do we expect users to emulate our patterns of dynamic datasources?

Reply:
(This comment is more directed at the base PR, which mixes dynamic data sources into example code)